### PR TITLE
Introduce earn section

### PR DESCRIPTION
**Pull request by artpi**
This PR is introducing earn section instead of Adwords section.

Functional changes:
- Rename WordAds tab to Earnings
- Earnings will be always visible (also for customers who cant have wordads)
- If you cant have wordads, we'll display a nudge

We need this for memberships.

zrzut ekranu 2019-02-26 o 13 52 46

## When you dont have WordAds enabled, but have a proper plan:

zrzut ekranu 2018-12-03 o 19 19 04

## When you don't have a proper plan

Previously, if you were on plan that does enable activating WordAds, the "WordAds" tab wasn't even visible. But this will tease people with the new WordAds tab and I am introducing a nudge for encouraging signing up for a higher plan.

zrzut ekranu 2019-02-28 o 17 35 44

### CODE REVIEW

File: client/my-sites/sidebar/sidebar.jsx [hunk 1]
```diff
@@ -199,28 +199,24 @@ export class MySitesSidebar extends Component {
...
-					tipTarget="wordads"
-				/>
-			);
-		}
-
-		return null;
+		return (
+			<SidebarItem
+				label={ translate( 'Earnings' ) }
+				selected={ itemLinkMatches( '/ads', path ) }
```
Comment (rclations) [inline client/my-sites/sidebar/sidebar.jsx]: If you switch `/ads` here to `/earn`, then the menu item will reflect when it's been selected.

**Comment by adamziel** [comment]
@artpi just so you're aware, we tried something along these lines, it linked to an upsell page like this one:

https://user-images.githubusercontent.com/205419/43253751-f10f65ae-90c5-11e8-9c33-8aa6a1c09275.png

We turned it off in the end but we didn't really have time to iterate on it and I think there's a lot of space for experimenting, so I'm excited to see how that goes! Would you mind keeping me in the loop? Some links for more context: p9jf6J-Hl-p2 p8Eqe3-sy-p2 https://github.com/Automattic/wp-calypso/pull/26301

Also, it's important to think about proper way of measuring success - funnels and hypotheses are... not fantastic for things like that. See the discussion here: https://wp.me/p9jf6J-Yt

File: client/my-sites/sidebar/sidebar.jsx (hunk 1 above)
Comment (artpi) [reply client/my-sites/sidebar/sidebar.jsx]: Switched to "Earn"

File: client/my-sites/earn/ads/main.jsx [hunk 2]
```diff
@@ -247,39 +221,18 @@ class AdsMain extends Component {
 			! isBusiness( site.plan )
 		) {
 			component = this.renderInstantActivationToggle( component );
+		} else if (
+			// ! site.options.wordads &&
```
[the remark this answers is not in the document]
Comment (artpi) [reply client/my-sites/earn/ads/main.jsx]: Done!

**Comment by artpi** [comment]
> Can we hide the tabs when this promo is shown?

That would be hard. On top of that, we are doing all this to have another tab here, with memberships. Hiding the tab will make no sense if there will be memberships as well.

>    * "Advanceddesign" -> "Advanced Design"
>    * "13GBstorage" -> "13GB Storage"
>    * "Upgrade to a Premium Plan" -> "Upgrade to the Premium Plan"

This is controlled across whole wpcom. I dont know why its truncated.

>    * "using WordAds programme." -> "using the WordAds program." (though I'm guessing this might be i18n as I also see the price in Euros)

Fixed

>    * "enroll into WordAds network" -> "enroll into the WordAds network"

Fixed

>    * "Earn from your content and traffic" -> "Earn money from your content and traffic"

Fixed.

**Comment by artpi** [comment]
- Rebased PR
- I updated the top screenshot
- Implemented feedback

@rclations could you have another look?

**Comment by cphilleo** [comment]
@rclations I think we need to discuss under what scenarios the instant activation toggle should show.  In production right now it's hidden for Atomic Business sites (presumably because they are using Jetpack), but it's shown for Jetpack Premium plans and hidden for Professional plans. Seems inconsistent.

@artpi There is a hard-coded link to the earning section from Settings -> Traffic -> Ads that will need to be updated.  You can find it here: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/jetpack-ads.jsx#L146

**Comment by artpi** [comment]
> There is a hard-coded link to the earning section from Settings -> Traffic -> Ads that will need to be updated

@cphilleo updated!

@davemart-in I'd appreciate another look here :)

File: client/my-sites/earn/ads/wrapper.jsx [hunk 3]
```diff
@@ -225,14 +179,33 @@ class AdsMain extends Component {
 		);
 	}
 
+	renderUpsell() {
+		const { translate } = this.props;
+		return (
+			<UpgradeNudgeExpanded
+				plan={ PLAN_PREMIUM }
+				title={ translate( 'Upgrade to a Premium Plan and start earning' ) }
```
Comment (torres126) [inline client/my-sites/earn/ads/wrapper.jsx]: > This is controlled across whole wpcom. I dont know why its truncated.

Does that still apply here? A lower case "p" would be consistent with capitalisation in running text, and so would "the". 

```suggestion
				title={ translate( 'Upgrade to the Premium plan and start earning' ) }
```

File: client/my-sites/earn/ads/wrapper.jsx [hunk 4]
```diff
@@ -225,14 +179,33 @@ class AdsMain extends Component {
...
+				subtitle={ translate(
+					"By upgrading to a Premium Plan you'll be able to monetize your site using WordAds program."
```
Comment (torres126) [inline client/my-sites/earn/ads/wrapper.jsx]: Same thing with the lower case "P" (consistency with other running text, for example, upgrade nudges)

```suggestion
					"By upgrading to the Premium plan, you'll be able to monetize your site using WordAds program."
```

**Comment by artpi** [comment]
> The same thing happens when a member who isn't an administrator tries to view that tab on a WPcom site. Currently, the WordAds tab doesn't appear, and in my opinion, that makes more sense in cases where the member isn't an admin

I wanted other stuff to be in this section as well - stuff that may have other access patterns.
But I changet this logic in https://github.com/Automattic/wp-calypso/pull/29001/commits/b75970617cb59ed99604cf39a9ecf67f0c942916 and it should be good now

**Comment by torres126** [comment]
Confirmed that other members of a site don't see the Earnings tab now. :tada:

Still feels broken with a JN site, but maybe that's something for the future.

**Pull request merged**